draft-ietf-quic-manageability.md

```diff
@@ -945,68 +945,72 @@ deliver packets between server and NAT. Reducing the timeout on UDP NATs might
-secretly changes its connection ID, there will be no mapping for the NAT, and
-the connection will suddenly break.
-
-QUIC is deliberately designed to fail rather than persist when the network
-cannot support its operation. For HTTP/3, this extends to recommending a
-fallback to TCP-based versions of HTTP rather than persisting with a QUIC
-connection that might be unstable. And {{?I-D.ietf-quic-applicability}}
-recommends TCP fallback for other protocols on the basis that this is preferable
-to sudden connection errors and time outs.
+Use of QUIC's connection ID for NAT mappings is not a viable approach to
+extend operational limits of deployed NATs, as the connection ID is not
+guaranteed to be constant over the lifetime of a connection. Endpoints may 
+begin using new connection IDs at will at any time during the connection.
+As such, the use of the connection ID
+to multiplex connections over a single address and port would cause the
+connectivity to break when the connection ID changes to a value that does not have
+an active mapping.
+
+Not-yet-used connection IDs are unknown to in-network devices and cannot
+be easily associated with a connection ID in use, as new connection IDs are
+negotiated inside cryptographically protected packets between the endpoints
+and as such are not observable in the network. Alternatively, sharing
+configurations between the client and NAT might be logistically challenging
+or impossible, especially when client migrates with an existing connection to
+a point behind the NAT. {{QUIC_LB}} described mechanisms to encode the client's
+identity in
+a connection ID to be used for load balancing. However, the described approach
+requires explicit coordination between the endpoint and network devices using the
+connection ID under the assumption that servers and load balancers are deployed
+by the same entity or cooperating entities.
+
+Note that multiplexing connection IDs over a single port anyway violates the
+best common practice to avoid "port overloading" as described in {{?RFC4787}}.
+
 Furthermore, wide deployment of NATs with this behavior hinders the use of
 QUIC's migration function, which relies on the ability to change the connection
 ID any time during the lifetime of a QUIC connection.
 
-It is possible, in principle, to encode the client's identity in a connection ID
-using the techniques described in {{QUIC_LB}} and explicit coordination with the
-NAT. However, this implies that the client shares configuration with the NAT,
-which might be logistically difficult. This adds administrative overhead
-while not resolving the case where a client migrates to a point behind the NAT.
-
-Note that multiplexing connection IDs over a single port anyway violates the
-best common practice to avoid "port overloading" as described in {{?RFC4787}}.
+If a NAT hits an operational limit where the available public IP addresses and
+ports are
+exhausted, it should drop the initial packets of a connection rather than
```

Comment:
NATs don't know about connections, they can only observe flows.

```suggestion
exhausted, it should drop the initial packets of a flow rather than
```